modules/virt-mounting-windows-driver-disk-on-vm.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-edit-vms.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-mounting-windows-driver-disk-on-vm_{context}"]
+
+= Mounting a Windows driver disk on a virtual machine
+
+You can mount a Windows driver disk on a virtual machine (VM) by using the {product-title} web console.
+
+.Procedure
+
+. In the *Administrator* view of the web console, navigate to *Virtualization* -> *VirtualMachines*.
```

Comment:
Hey, IIUC, you mention here that only admins can add the drive. Actually, not only admins can add the drive; any VM owner can do it.